app/route_rangers_api/utils/city_mapping.py

```diff
@@ -1,5 +1,5 @@
 CITIES_CHOICES = {"CHI": "Chicago", "NYC": "New York", "PDX": "Portland"}
-CITIES_CHOICES_SURVEY = {"Chicago":"CHI","NewYork":"NYC","Portland":"PDX"}
+CITIES_CHOICES_SURVEY = {"Chicago": "CHI", "NewYork": "NYC", "Portland": "PDX"}
```

Comment:
seems like we could build this into the city context dict below since they are keyed the same way, but fine for the time being